lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -1390,6 +1390,25 @@ class DecomposeAtenWhereScalarSelfOp
 };
 } // namespace
 
+// Decompose aten.masked_fill.Scalar into aten.where.self op.
+namespace {
+class DecomposeAtenMaskedFillScalarOp
```

Comment:
I think we can remove the linalg lowering for this op now that we have the decomposition